src/layouts/DashboardLayout.vue

```diff
@@ -120,7 +120,7 @@
               <p class="text-newlines" v-if="availableUpdate.notes">
                 {{ availableUpdate.notes }}
               </p>
-              <b-alert variant="warning" show>
+              <b-alert variant="warning" show v-if="0 < numChannels">
```

Comment:
Just a tiny nit to refactor with code consistency (variable on the left of the conditional)

```suggestion
              <b-alert variant="warning" show v-if="numChannels > 0">
```

Or just:

```suggestion
              <b-alert variant="warning" show v-if="numChannels">
```